src/routes/activityReports/handlers.js

```diff
@@ -120,7 +120,9 @@ export async function submitReport(req, res) {
 }
 
 export async function getActivityRecipients(req, res) {
-  const activityRecipients = await possibleRecipients();
+  const { region } = req.query;
+  const targetRegion = region ? parseInt(region, DECIMAL_BASE) : undefined;
```

Comment:
We need to verify that the user does in fact have access to the region being passed in. The authorization filtering on the client side is good, but is insufficient for maintaining a secure system.

Reply:
Based on https://gsa-tts.slack.com/archives/C017RJ4GEET/p1614609580004100?thread_ts=1614606923.002700&cid=C017RJ4GEET the existing authorization checking for login & SITE_ACCESS is sufficient